catalogue/webapp/components/ExpandedImage/ExpandedImage.tsx

```diff
@@ -308,9 +308,9 @@ const ExpandedImage: FunctionComponent<Props> = ({
         });
 
   return (
-    <div role="dialog" aria-labelledby={ariaLabelledBy} aria-modal={true}>
+    <div role="dialog" id={ariaLabelledBy} aria-modal={true}>
       <Overlay onClick={() => setExpandedImage(undefined)} />
-      <Modal ref={modalRef}>
+      <Modal ref={modalRef} aria-labelledBy={ariaLabelledBy}>
```

Comment:
Don't think we're using aria-labelledBy correctly here. The value should be the id of an element(s) that provide the text needed for labeling. I don't think the dialog div does that. See [using aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute)

Reply:
Also, aria-labelledby should all be lowercase

Reply:
gotcha, i think i will remove this. thanks for this.